pkg/sql/plan2/explain/init.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2021 - 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package explain
+
+//BuiltinFunctionsMap  change BuiltinFunctions to map
+var SysFunsAndOperatorsMap map[string]FunctionProtoType
+
+func init() {
+	//change BuiltinFunctions to map
+	SysFunsAndOperatorsMap = make(map[string]FunctionProtoType)
+	for _, fun := range BuiltinFunctions {
+		SysFunsAndOperatorsMap[fun.Name] = fun
+	}
+}
+
+type FunctionKind int32
+
+const (
+	STANDARD_FUNCTION          FunctionKind = 0
```

Comment:
What do you need all these?    We have too many function files already, explain should not need this, ask mingsong or yuanning, work it out in one place.

Reply:
OK , Then I'll change it in the functions.go file